test_scripts/Smoke/API/028_AlertManeuver_PositiveCase.lua

```diff
@@ -0,0 +1,167 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: AlertManeuver
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [AlertManeuver] SUCCESS on Navigation.AlertManeuver
+--
+-- Description:
+-- Mobile application sends AlertManeuver request with valid parameters to SDL
+--
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+--
+-- Steps:
+-- Application sends AlertManeuver request with valid parameters to SDL
+--
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if Navigation interface is available on HMI
+-- SDL checks if TTS interface is available on HMI
+-- SDL checks if AlertManeuver is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the Navigation part of request with allowed parameters to HMI
+-- SDL transfers the TTS part of request with allowed parameters to HMI
+-- SDL receives Navigation part of response from HMI with "SUCCESS" result code
+-- SDL receives TTS part of response from HMI with "SUCCESS" result code
+-- SDL transfers response to mobile app
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+
+--[[ Local Variables ]]
+local putFileParams = {
+  requestParams = {
+    syncFileName = 'icon.png',
+    fileType = "GRAPHIC_PNG",
+    persistentFile = false,
+    systemFile = false
+  },
+  filePath = "files/icon.png"
+}
+
+local requestParams = {
+  ttsChunks = {
+    {
+      text = "FirstAlert",
+      type = "TEXT",
+    },
+    {
+      text = "SecondAlert",
+      type = "TEXT",
+    },
+  },
+  softButtons = {
+    {
+      type = "BOTH",
+      text = "Close",
+      image = {
+        value = "icon.png",
+        imageType = "DYNAMIC",
+      },
+      isHighlighted = true,
+      softButtonID = 821,
+      systemAction = "DEFAULT_ACTION",
+    },
+    {
+      type = "BOTH",
+      text = "AnotherClose",
+      image = {
+        value = "icon.png",
+        imageType = "DYNAMIC",
+      },
+      isHighlighted = false,
+      softButtonID = 822,
+      systemAction = "DEFAULT_ACTION",
+    },
+  }
+}
+
+local responseNaviParams = {}
+responseNaviParams.softButtons = commonFunctions:cloneTable(requestParams.softButtons)
+for k, _ in pairs(responseNaviParams.softButtons) do
```

Comment:
@GetmanetsIrina I would propose to create separate local function for preparation test data `responseNaviParams` with this loop in it instead of global code in test script. This will improve readability of script.

Reply:
:+1:

Reply:
@aderiabin , Done in f892e14586f34b582ee7c5fe4d0abb9ab152063e.